api/WebTransportSendStream.json

```diff
@@ -0,0 +1,103 @@
+{
+  "api": {
+    "WebTransportSendStream": {
+      "__compat": {
+        "spec_url": "https://w3c.github.io/webtransport/#webtransportsendstream",
+        "support": {
+          "chrome": {
+            "version_added": false
+          },
+          "chrome_android": "mirror",
+          "edge": "mirror",
+          "firefox": {
+            "version_added": "114"
```

Comment:
This is correct too. Added in https://bugzilla.mozilla.org/show_bug.cgi?id=1811958